lib/WiFi/src/WiFiMode.cpp

```diff
@@ -138,8 +157,20 @@ esp_err_t WiFiMode::start()
 
 esp_err_t WiFiMode::eventHandler(void* ctx, system_event_t* event)
 {
-    //WiFiMode wifi_mode = reinterpret_cast<WiFiMode*>(ctx);
-    return ESP_OK;
+    ESP_LOGD(TAG, ">> *****************EVENT HANDLER CALLED*******************************************");
+    ESP_LOGD(TAG, ">> ********************************************************************************");
+    WiFiMode* pWiFiMode = reinterpret_cast<WiFiMode*>(ctx);
```

Comment:
Lowercase variable name according to style guide, maybe `wifi_mode` (without [hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation))